packages/LSOnline/modules/database/Database.js

```diff
@@ -0,0 +1,49 @@
+// Load sequelize
+const Sequelize = require('sequelize')
+const Logger = require('../utils/Logger')
+const db = {}
+
+// Create connection
+const sequelize = new Sequelize('game', 'homestead', 'secret', {
+  host: '192.168.10.10',
+  dialect: 'mysql',
+  operatorsAliases: false,
+
+  pool: {
+    max: 5,
+    min: 0,
+    acquire: 30000,
+    idle: 10000
+  }
+})
+
+// Test database connection
+sequelize
+  .authenticate()
+  .then(() => {
+    Logger.info('[Database] Connection has been established successfully.')
+  })
+  .catch(err => {
+    Logger.fatal('[Database] Unable to connect to the database!')
+  })
+
+// Assign sequelize to 'db'
+db.sequelize = sequelize
```

Comment:
tu jest troche fuckup, zamiast sequelize może nazwać to połączenie, skoro wyżej jest zanotowane, że te połączenie jest tworzone.